pavelib/utils/envs.py

```diff
@@ -271,6 +274,22 @@ def get_django_settings(cls, django_settings, system, settings=None):
                 print(f.read())
             sys.exit(1)
 
+    @classmethod
+    def get_django_json_settings(cls, django_settings, system, settings=None):
+        """
+        Interrogate Django environment for specific settings value
+        :param django_settings: list of django settings values to get
+        :param system: the django app to use when asking for the setting (lms | cms)
+        :param settings: the settings file to use when asking for the value
+        :return: json value of the django setting
```

Comment:
Does this return a string or a decoded JSON object?

Reply:
This returns a string, since it changes the `True` values to `true` which would be invalid in a Python object.

I'll update the docstring to mention it in a better way. 👍🏼

Reply:
Addressed 👍🏼 

Updated to 
```
        :return: json string value of the django setting
```